pkg/sql/alisa_submitter.go

```diff
@@ -48,8 +138,58 @@ func (s *alisaSubmitter) ExecuteExplain(cl *ir.ExplainStmt) error {
 	return fmt.Errorf("Alisa submitter does not support EXPLAIN clause")
 }
 
-func (s *alisaSubmitter) Setup(w *pipe.Writer, db *database.DB, modelDir string, cwd string, session *pb.Session) {
+func (s *alisaSubmitter) GetTrainStmtFromModel() bool { return false }
+func init()                                           { SubmitterRegister("alisa", &alisaSubmitter{&defaultSubmitter{}}) }
+
+func (s *alisaSubmitter) achieveResource(entryCode, tarball string) error {
+	if err := writeFile(filepath.Join(s.Cwd, entryFile), entryCode); err != nil {
+		return err
+	}
+
+	path, err := findPyModulePath("sqlflow_submitter")
+	if err != nil {
+		return err
+	}
+	cmd := exec.Command("cp", "-r", path, ".")
+	cmd.Dir = s.Cwd
+	if _, err := cmd.CombinedOutput(); err != nil {
+		return fmt.Errorf("failed %s, %v", cmd, err)
+	}
+
+	cmd = exec.Command("tar", "czf", tarball, "./sqlflow_submitter", entryFile)
```

Comment:
maybe replace with https://golang.org/pkg/archive/tar/ later.

Reply:
Good point, I would not use the `achive` package here just because the example code is too long: https://golang.org/pkg/archive/tar/#example__minimal
I will implement some helper function of `achive` in the future so that the `alisaSubmitter` code can be more clearly.